transcription/transcription.go

```diff
@@ -0,0 +1,31 @@
+package transcription
+
+import (
+	"net/smtp"
+	"strings"
+)
+
+// SendEmail authenticates, creates message, and sends email using arguments.
```

Comment:
Can we fix up the wording of this comment. Authenticates what? Creates message -> create messages or creates a message. Using arguments is probably unnecessary.